docs/source/install/install.rst

```diff
@@ -91,13 +91,10 @@ Package names are different depending on your CUDA Toolkit version.
 
 .. note::
 
-   Wheel packages are built with NCCL (Linux only) and cuDNN support enabled.
+   To enable features provided by additional CUDA libraries (cuTENSOR / NCCL / cuDNN), you need to install them manually.
+   If you installed CuPy via wheels, you can use the installer command below to setup these libraries.
```

Comment:
```suggestion
   If you installed CuPy via wheels, you can use the installer command below to setup these libraries in case you don't have a previous installation.
```